exercises/diffie-hellman/example.js

```diff
@@ -0,0 +1,29 @@
+export default class DiffieHellman {
+
+  constructor(p, g) {
+    if (p < 2 || g < 2 || p > 7919 || g > 7919) {
```

Comment:
Does `7919` the last prime used in `PRIMES`? Why not use `PRIMES[PRIMES.length - 1]` instead of a hardcoded value?